client-js/app/elements/labrad-plot.ts

```diff
@@ -274,7 +277,17 @@ export class Plot extends polymer.Base {
    */
   attached(): void {
     this.render();
-    window.addEventListener('resize', (e) => this.resizePlot());
+    this.lifetime.defer(() => { this.finishRender = true; });
+    const resizeListener = (e) => this.resizePlot();
+    window.addEventListener('resize', resizeListener);
+    this.lifetime.defer(() => {
```

Comment:
Yes, could do that. The reason I prefer doing it this way is to follow the way go's defer statement is used: do operation A then defer the cleanup of A; next do operation B and defer the cleanup of B. That way each setup/cleanup pair is small and self-contained.